frontend/src/pages/meditation/libs/types/entry.type.ts

```diff
@@ -0,0 +1,7 @@
+type Entry = {
```

Comment:
give more meaningful name
```suggestion
type MeditationEntry = {
```

Reply:
Done